lib/evss/disability_compensation_auth_headers.rb

```diff
@@ -37,7 +37,7 @@ def gender
       when 'M'
         'MALE'
       else
-        Raven.extra_context(user_gender: @user.gender)
+        Raven.extra_context(user_gend: @user.gender)
```

Comment:
Was this intentional? Just making sure this wasn't a typo, otherwise I'll approve

Reply:
It's the same header as in the spec that uses it, so I'm pretty sure it's intentional.

Reply:
yeah, maybe it's a bit sketchy of me, but I'm trying to get around the PII sanitizer.  I figure if we're logging this string and nothing else it's not a violation of anyone's privacy.